101-multiple-windows-vms-with-common-script/azuredeploy.json

```diff
@@ -0,0 +1,299 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+
+        "vmCount": {
+            "type": "int",
+            "defaultValue": 2,
+            "metadata": {
+                "description": "Number of LAB-VM tobe deployed"
+            }
+        },
+        "dns-prefix": {
+            "type": "string",
+            "defaultValue": "labvm-400",
+            "metadata": {
+                "description": "DNS Prefix used by LabVM Public IP"
+            }
+        },
+        "adminUser": {
+            "type": "string",
+            "defaultValue": "mahendra",
+            "metadata": {
+                "description": "Admin Username for Lab VM"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Password for admin user"
+            }
+        },
+        "vmsize": {
+            "type": "string",
+            "defaultValue": "Standard_D4s_v3",
+            "metadata": {
+                "description": "VM Size for all lab-vms. "
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location to deploy current resource"
+            }
+        },
+
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located including a trailing '/'"
+            },
+            "defaultValue": "[deployment().properties.templateLink.uri]"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "defaultValue": "",
+            "metadata": {
+                "description": "SAS Token for accessing script path"
+            }
+        },
+        "scriptFilename": {
+            "type": "string",
+            "defaultValue": "jenkins-java8.ps1",
+            "allowedValues": [
+                "jenkins-java8.ps1",
+                "az-500.ps1",
+                "container-lab.ps1",
+                "container-vs2019-lab.ps1"
+            ],
+            "metadata": {
+                "description": "Choose script to launch from list of available scripts"
+            }
+        }
+    },
+    "functions": [],
+    "variables": {
+        "strname": "[toLower(concat('labvmstg',uniqueString(resourceGroup().id)))]",
+        "scriptUrl": "[uri(parameters('_artifactsLocation'),concat(parameters('scriptFilename'),parameters('_artifactsLocationSasToken')))]"
+    },
+    "resources": [
+        {
+            "name": "[variables('strname')]",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
done